### ISSUE TITLE
HLS chunks files extension

### ISSUE THREAD

**Issue by mirkos93** [pascalbaljetmedia/laravel-ffmpeg#174]
Is it possible to change the chunks filename pattern to something customized?

Thanks

**Comment by pascalbaljet** [comment]
I've rewritten the package and support for custom patterns is included. Could you test the `v7` branch? Almost no breaking changes except for some filter stuff in HLS exports.

https://github.com/pascalbaljetmedia/laravel-ffmpeg/tree/v7

`composer require "pbmedia/laravel-ffmpeg:v7.x-dev"`

```php
<?php

FFMpeg::fromDisk('videos')
    ->open('video.mp4')
    ->exportForHLS()
    ->useSegmentFilenameGenerator(function ($name, $format, $key, callable $segments, callable $playlist) {
        $segments("{$name}-{$format}-{$key}-%03d.ts");
        $playlist("{$name}-{$format}-{$key}.m3u8");
    });
```

**Comment by pascalbaljet** [comment]
Fixed in `v7.0`
